data/news.csv

```diff
@@ -0,0 +1,3 @@
+Date,Text
+2022-10-07,Initial publication of the website.
+2022-10-10,Added hex sticker for the working group. Credits to <a href='https://www.flaticon.com/free-icons/technology' title='technology icons'>Kalashnyk</a> on Flaticon for creating the sticker icon.
```

Comment:
Is this robust when there are commas in the news items themselves?

Reply:
It should now – I realised that editing the .csv with Excel was not a good idea, and needed to use a text editor instead!